### ISSUE TITLE
Input Fields Accessibility

### ISSUE THREAD

**Issue by Cadranel-Kael** [robsontenorio/mary#882]
Hi! I’m really loving MaryUI, it’s awesome to have an open-source alternative to FluxUI.
I noticed that the input fields are wrapped in `<fieldset>` elements. According to the W3C spec, “The `<fieldset>` element represents a set of form controls optionally grouped under a common name.” ([W3C Spec](https://www.w3.org/TR/2011/WD-html5-author-20110809/the-fieldset-element.html)).
I was wondering: why use a `<fieldset>` instead of a `<div>` or `<label>`? Also, `<legend>` elements don’t have the built-in functionality to focus the input when clicked, unlike `<label>` elements.
Furthermore, when using `<x-input>`, I noticed that there are two `<label>` elements surrounding the input.
Thanks again for your amazing work on MaryUI!

**Comment by robsontenorio** [comment]
Yes , because I have mirrored daisyUI examples.

**Comment by brun0f0nseca** [comment]
Hi, I also have the same problem. I have tried everything to fix it and cannot seem to "escape" this error. I have tried changing the `aria-label` to fix it, but it is impossible. I keep getting the same warning about it...

`<x-mary-select aria-label="{{ __('public.travel-simulator.from') }}" label="{{__('public.travel-simulator.from')}}" wire:model="from" :options="$itineraryStopsName" placeholder="{{__('public.travel-simulator.select-from')}}" class="w-32! p-0! bg-none [&_select]:appearance-auto" required />`

<img width="985" height="512" alt="Image" src="https://github.com/user-attachments/assets/97c3fe5e-f1ba-422f-941e-ef0ebd8ecaaf" />
